nightingale_utils/configs/robot_configuration_lookup.yaml

```diff
@@ -3,40 +3,101 @@ joint_configurations: {
         head: {
             names: ["pan_joint", "tilt_joint"],
             joints: [0.0,0.0],
+            cartesian: [
+                [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
         },
         torso: {
             names: ["linear_joint"],
             joints: [0.1],
+            cartesian: [
+                [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
         },
         left_arm: {
             names: ["left_shoulder_pan_joint", "left_shoulder_lift_joint", "left_arm_half_joint", "left_elbow_joint", "left_wrist_spherical_1_joint", "left_wrist_spherical_2_joint", "left_wrist_3_joint"],
             joints: [1.6285100157139785, 1.4853564921068, -0.4129526423290365, 2.680378098157231, 0.03587460652925101, -0.45225347904488444, 1.7006671448238166],
+            cartesian: [
+                [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
```

Comment:
There should be a position and orientation (quaternion) array here instead of a matrix.